skimage/data/__init__.py

```diff
@@ -754,3 +756,33 @@ def lfw_subset():
 
     """
     return _np.load(_os.path.join(data_dir, 'lfw_subset.npy'))
+
+
+def allen_mouse_brain_atlas():
+    """
+    The Allen Institute's averaged mouse brain atlas at 100 micron isotropic resolution, grayscale, normalized to MAD, zero-padded by 5.
+    
+    Returns
+    -------
+    allen_mouse_brain_atlas : (142, 90, 124) float64 ndarray
+
+    Notes
+    -----
+    © 2015 Allen Institute for Brain Science. Allen Mouse Brain Atlas. 
+    Available from: http://download.alleninstitute.org/informatics-archive/current-release/mouse_ccf/average_template/
+    """
+
+    return _np.load(_os.path.join(data_dir, 'allen_mouse_brain_atlas.npy'))
+
+
+def cleared_mouse_brain():
+    """
+    A clarity-cleared mouse brain at 100 micron isotropic resolution, grayscale, normalized to MAD, zero-padded by 5.
+    
+    Returns
+    -------
+    cleared_mouse_brain : (142, 83, 121) float64 ndarray
```

Comment:
what is the origin and license of the data?

Reply:
This is a downsampled and processed image generated at Karl Deisseroth's lab at Stanford University, we have been given permission to share it. Do you need something more specific like for it to be hosted at some url with a particular license?